README.md

```diff
@@ -0,0 +1,86 @@
+# Ansys actions
+
+[![ansys][Ansys badge]][actions.docs.ansys.com]
+[![CI-CD][CI-CD badge]][CI-CD yml]
+[![MIT][MIT badge]][MIT url]
+
+A repository containing a collection of [GitHub Workflows] to be reused by
+projects in the Ansys ecosystem.
+
+For more information on available actions and how to use them, see
+[actions.docs.ansys.com].
+
+## Recommended workflow strategy
+
+The different actions provided by [ansys/actions] can be used to create a
+simple but effective workflow.
+
+The following lines describe the suggested workflows for various events, such as
+pushing a new commit to a pull request, merging a commit to the main branch of a
+repository, and performing a new release. Each image showcases the distinct job
+steps declared in a YML file. Jobs highlighted in green signify that they
+execute, while those with a grey background indicate that they do not execute.
```

Comment:
```suggestion
steps declared in a YML file. Jobs highlighted in green signify that they
execute, while those with a blue background indicate that they do not execute.
```
Despite this proposed change, I wonder if we shouldn't update the doc to work with both light and dark modes. Currently green and blue are easy to identify in light mode but that's not the case at all in dark mode. Could you try to update the colors to darken the blue one and changes the documentation references (green and blue) accordingly ?
Since @Revathyvenugopal162 mentioned colors, would it be possible to pick a light color and dark color so that references can make sense in both dark and light modes ? I just checked